src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -0,0 +1,1560 @@
+            result_wr = write(sockfd, availableFileList.c_str(), availableFileList.size());
+
+            if(result_wr < 0) {
+                triangleCount_logger.log("Error writing to socket", "error");
+            }
+
+            triangleCount_logger.log("Sent : Available File List " + availableFileList, "info");
+
+            bzero(data, 301);
+            read(sockfd, data, 300);
+            response = (data);
+            response = utils.trim_copy(response, " \f\n\r\t\v");
+        }
+
+        if (response.compare(JasmineGraphInstanceProtocol::OK) == 0) {
+            triangleCount_logger.log("Received : " + JasmineGraphInstanceProtocol::OK, "info");
+
+            std::vector<std::string> chunksVector;
+
+            for (unsigned i = 0; i < compositeCentralStoreFileList.length(); i += INSTANCE_DATA_LENGTH - 10) {
+                std::string chunk = compositeCentralStoreFileList.substr(i, INSTANCE_DATA_LENGTH - 10);
+                if (i + INSTANCE_DATA_LENGTH - 10 < compositeCentralStoreFileList.length()) {
+                    chunk += "/SEND";
+                } else {
+                    chunk += "/CMPT";
+                }
+                chunksVector.push_back(chunk);
+            }
+
+            for (int loopCount = 0; loopCount < chunksVector.size(); loopCount++) {
+                if (loopCount == 0) {
+                    std::string chunk = chunksVector.at(loopCount);
+                    write(sockfd, chunk.c_str(), chunk.size());
+                } else {
+                    bzero(data, INSTANCE_DATA_LENGTH);
+                    read(sockfd, data, INSTANCE_DATA_LENGTH);
+                    string chunkStatus = (data);
+                    std::string chunk = chunksVector.at(loopCount);
+                    write(sockfd, chunk.c_str(), chunk.size());
+                }
+            }
+
+            bzero(data, 301);
+            read(sockfd, data, 300);
+            response = (data);
+            response = utils.trim_copy(response, " \f\n\r\t\v");
+        }
+
+        if (response.compare(JasmineGraphInstanceProtocol::OK) == 0) {
+            triangleCount_logger.log("Received : " + JasmineGraphInstanceProtocol::OK, "info");
+            result_wr = write(sockfd, std::to_string(threadPriority).c_str(), std::to_string(threadPriority).size());
+
+            if(result_wr < 0) {
+                triangleCount_logger.log("Error writing to socket", "error");
+            }
+
+            triangleCount_logger.log("Sent : Thread Priority " + std::to_string(threadPriority), "info");
+
+            bzero(data, 301);
+            read(sockfd, data, 300);
+            response = (data);
+            response = utils.trim_copy(response, " \f\n\r\t\v");
+            string status = response.substr(response.size() - 5);
+            std::string result = response.substr(0, response.size() - 5);
+
+            while (status == "/SEND") {
```

Comment:
We should change this to `status.compare("/SEND") == 0`

Reply:
I changed it as requested